<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QWidget" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>160</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Password:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Email:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Login:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="leLogin"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lePassword"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="leEmail"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnAuth">
   <property name="geometry">
    <rect>
     <x>84</x>
     <y>180</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Authorization</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnChange">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>180</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Change</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
